### ISSUE TITLE
Multiple ::file-selector-button Issue

### ISSUE THREAD

**Issue by Woedenaz** [postcss/autoprefixer#1419]
Hey there! I'm having an issue with the latest release.

So, let's say I have:
```css
input::file-selector-button {
	color: black;
}

input:hover::file-selector-button {
	color: white;
}
```

When it is run through autoprefixer (with any browserslist, as far as I can tell), it results in:

```css
input:hover::-webkit-file-upload-button {
	color: white;
}

input:hover::file-selector-button {
	color: white;
}

input:hover::file-selector-button {
	color: white;
}
```

Sometimes it's just the first two, depending on the browsers list. Regardless, it always removes anything before the last instance of ::file-selector-button. Even if I have two different selections.

For instance, if I have:
```css
input::file-selector-button {
	color: black;
}

div > input::file-selector-button {
	color: black;
}
```

it will output:

```css
div > input::-webkit-file-upload-button {
	color: black;
}

div > input::file-selector-button {
	color: black;
}

div > input::file-selector-button {
	color: black;
}
```

I've tried everything I can think of to circumvent this but I'm unsure what to do at this point. Any help would be most appreciated!

**Comment by ai** [comment]
Autoprefixer Yeap. We have a strange bug. I will try to look during next week.

**Comment by usmanyunusov** [comment]
PR: https://github.com/postcss/autoprefixer/pull/1420

**Comment by ai** [comment]
The fix was released in 10.3.2
